### ISSUE TITLE
Tabby opens a new window when trying to open a new tab

### ISSUE THREAD

**Issue by fiction3** [Eugeny/tabby#5736]
Version: 1.0.171
Platform: Windows x64 10.0.17763
Plugins: none
Frontend: xterm

For some unknown reason Tabby opens a new window when trying to open a new tab.
Any help to resolve this is highly appreciated.

![image](https://user-images.githubusercontent.com/73464663/153705227-43f0d16c-b04b-4a85-b7f9-f0309744293d.png)

**Comment by Eugeny** [comment]
Could you check whether the `Use ConPTY` option in Settings -> Shell helps?

**Comment by fiction3** [comment]
Yes, it was enabled, though when disabling it works but the prompt is showing some strange character after.
![tabby](https://user-images.githubusercontent.com/73464663/153714957-f18bd58a-b372-4ccb-b458-4c43657a6416.png)
